methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+puts 'this is a string to practice with'
```

Comment:
Hey Igor, can you put this string in a variable and then call the methods below on that variable?

    practice_string = 'this is a string to practice with'
    puts practice_string
    puts practice_string.capitalize
    ....